ihmc-java-toolkit/src/main/java/us/ihmc/tools/property/StoredPropertySet.java

```diff
@@ -494,53 +489,12 @@ else if (key instanceof BooleanStoredPropertyKey booleanKey)
             }
          });
       }
-      else if (iniResourceExists()) // fallback to the old INI format
```

Comment:
I removed legacy INI support. It's been years and everything has been auto converted at this point